src/components/app/MenuButtons/MetaInfo.css

```diff
@@ -68,13 +68,16 @@
   font-weight: 700;
 }
 
-.moreInfoSummary::marker {
-  content: '';
+.moreInfoSummary {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 24px;
+  user-select: none;
 }
 
-.moreInfoButton {
```

Comment:
I removed the button because the summary already reacts to clicks. But I had to add quite a lot of CSS so that it looks like a button.

Reply:
That was my reason to using the button